hazelcast-jet-core/src/test/java/com/hazelcast/jet/pipeline/StreamStageTest.java

```diff
@@ -1253,6 +1253,39 @@ public void hashJoin() {
         );
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
+    public void innerHashJoin() {
```

Comment:
This seems to test the behavior common to inner and outer join. The input has no nulls to begin with.